arches/management/commands/updateproject.py

```diff
@@ -173,6 +174,28 @@ def update_to_v7_6(self):
                             os.path.join(dirpath, filename[:-7] + ".py"),
                         )
 
+        if os.path.isfile(os.path.join(settings.APP_ROOT, "apps.py")):
```

Comment:
this will need to be hardened, as `updateproject` calls all  methods in this file. So conceivably it could get called in a future iteration where `is_arches_application=True` has already been added

Reply:
Sorry, I'm not quite following. If apps.py exists in a future iteration, or even today (in the case of arches-templating), this just logs a UserWarning telling the user what to do. Do you mean change the wording of the UserWarning?

Reply:
ah sorry, I mean read the file && check if `is_arches_application=True` already exists. Someone could feasibly make this change to move from 7.5 -> 7.6, but if we release a 7.6.1 that requires running `updateproject` again they'd still get a warning.

Reply:
I think I see @chrabyrd's point, but at the same time it's very unlikely that we would ask a user to run updateproject on a patch release.

Reply:
True and fair @chiatt , but then how about an 8.0 release? Presumably `updateproject` will be run, and it if we follow the current pattern it will run all functions declared here.

Also open to a discussion around changing the pattern of course.

Reply:
Good point, but I should point out that this method is named `update_to_v7_6`. Version 8.0 is still a long way off and generally the updateproject file changes significantly between feature releases, let alone a major one.

Reply:
We expect people to be upgraded to the last feature release before an upgrade. So we can remove past upgrade methods and probably should.

Reply:
Just be clear, you're saying that you expect this pattern to change? https://github.com/archesproject/arches/blob/dev/7.6.x/arches/management/commands/updateproject.py#L26

and that we _don't_ just tack `self.update_to_v8_0()` to the end of it? If so, cool and I'm on board 👍

Reply:
ah -- once again you beat me to the comment 🙂

Reply:
> We expect people to be upgraded to the last feature release before an upgrade. So we can remove past upgrade methods and probably should.

Resolved in #11176